notebooks/ch14/bias_random_vars.md

```diff
@@ -65,26 +65,43 @@ plt.title('PMF of $X$');
 ![png](bias_random_vars_files/bias_random_vars_5_0.png)
 
 
-### Expectation
+### Joint Distributions
 
-We are often interested in the long-run average of a random variable because it gives us a sense of the center of the variable's distribution. We call this long-run average the **expected value**, or **expectation** of a random variable. The expected value of a random variable $ X $ is defined as:
+The notion of PMFs for single random variables extends naturally to joint distributions for multiple random variables; in particular, the *joint distribution* of a set of random variables yields the probability that they simultaneously take on a specific set of values.
 
-$$\mathbb{E}[X] = \sum_{x\in \mathbb{X}} x \cdot \mathbb{P}(X = x)$$
+As an example, let the random variable $ X $ represent the number of heads in 10 coin flips, and let $ Y $ represent the number of tails in the same set of 10 coin flips. We can note that $P(X=0, Y=10) = P(X=10, Y=0) = (0.5)^{10}$. Meanwhile $P(X=6, Y=6) = 0$ since we cannot possibly have 6 heads and 6 tails in 10 coin flips.
 
-For example, if $ X $ represents the roll of a single fair six-sided die, 
+#### Marginal Distributions
+
+Sometimes, we have the joint distribution for two random variables, but we want to find the *marginal distribution* for $X$. In other words, we would like to find the PMF of $X$ given the joint distribution for $X$ and $Y$. To find the probability that $X$ takes on a particular value, we must consider all possible values of $Y$ that can simultaneously happen with $X$ and sum over all of these joint probabilities:
```

Comment:
> marginal distribution* for $X$

maybe marginal distribution for one of the random variables instead? or state that our two RV's are X and Y